src/search.c

```diff
@@ -170,10 +238,20 @@ static void print_final_results(void)
 		b = ptr->data;
 		ptr = ptr->next;
 		counter++;
-		printf("Bundle %s\t(%li Mb)%s\n",
-		       b->bundle_name,
-		       b->size,
-		       b->is_tracked ? "\tinstalled" : "");
+		char *brstr;
+		char *szstr;
+		string_or_die(&brstr, "Bundle %s\t%s",
+			      b->bundle_name,
+			      b->is_tracked ? "[installed]\t" : "");
+		string_or_die(&szstr, "(%li Mb%s)",
+			      b->size / 1024 / 1024, /* convert from bytes->KB->MB */
+			      b->is_tracked ? " on system" : " to install");
+		/* do not print the size information when the scope is only one bundle ('o')
+		 * because we did not load all bundles and therefore do not have include sizes
+		 * for the result */
+		printf("%s%s\n", brstr, scope != 'o' ? szstr : "");
+		free_string(&brstr);
+		free_string(&szstr);
```

Comment:
Optional: Consider using multiple printf calls. Libc buffers stdout so I guess in practice the result will be the same.

```
    printf("Bundle ...", ...); /* No \n. */
    if (scope != 'o') {
        printf("(%li Mb...)\n", ...);
    } else { /* Or even drop the \n above and this else, always doing putchar... */
        putchar('\n');
    }
```